src/view/home.js

```diff
@@ -92,7 +93,23 @@ const balanceStyles = StyleSheet.create({
     marginTop: 30,
     marginBottom: 5,
   },
-});
+};
+
+const balanceStyles = createStyles(
+  baseBalanceStyles,
+
+  maxWidth(breakWidth, {
+    wrapper: {
+      width: 350,
+    },
+  }),
+
+  maxWidth(smallBreakWidth, {
+    wrapper: {
+      width: 250,
+    },
+  })
+);
```

Comment:
Why does the wrapper need a width?

Reply:
Just tested removing the diff on home.js. It didn't change anything.

Reply:
I get this UI bug when I don't set the width in the wrapper:

![51401843_294491867927029_6915343391798591488_n](https://user-images.githubusercontent.com/10647624/52388671-5ee94780-2a44-11e9-955d-b3c057892084.jpg)

Reply:
I think it's because `adjustFontSizeToFit` needs a reference point to know what font size to use?

Reply:
@valentinewallace I think the issue with the btc unit behind the balance is causing `adjustFontSizeToFit` to fail. If you display then number as fiat it scales as intended.

Reply:
Ah okay, so are you saying it's ok to leave the `width` as is?